components/Footer.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export default function Footer(): JSX.Element {
+  return <StyledFooter></StyledFooter>;
```

Comment:
Usually we try to only commit code that is already useful for the user. Empty you could just leave uncommited until you fill them with content :)

Reply:
Before I pushed, it said 'Footer' 😅

But yeah, I see the point!